### PR TITLE
Walkthrough on-site

### PR DESCRIPTION
**Pull request by slavingia**

### CODE REVIEW

**Comment by slavingia** [comment]
#### 📝 Changed routes:


| [**/**](https://fweb3-1c1ugke5l-shlvc.vercel.app/) |
|:-:|
|<a href="https://fweb3-1c1ugke5l-shlvc.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/FbNHqKQd68b4rWj0azq9R6aOSXfmKxHXd-l9URjzA0j2pi5ArFPXD_CXxno0qLG3V57KuTJsXFRrfAjoJH0IKg.png">(view full size)</a> |







Commit 55aeaf887af9286fd412ce289f72a189122879d7 (https://fweb3-1c1ugke5l-shlvc.vercel.app).

File: pages/index.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { useEffect, useState } from "react";
 import { injected } from "../connectors";
 import useMetaMaskOnboarding from "../hooks/useMetaMaskOnboarding";
 import * as React from 'react'
```
Comment (slinlee) [inline pages/index.tsx]: It will be nice to combine this import with the one above from react

**Comment by slavingia** [comment]
@slinlee @epan lmk what y'all think!

File: cypress/integration/walkthrough.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+// check out the link below and learn how to write your first test:
+// https://on.cypress.io/writing-first-test
+
+describe("Walkthrough Tests", () => {
+  it("should load data for each dot", () => {
+    cy.visit("/");
+
+    cy.get("h2").should("contain.text", "Learn and build in web3.");
+
+    cy.get(".js-dot0").click();
```
Comment (slinlee) [inline cypress/integration/walkthrough.js:14]: looks like the tests need to be updated 
image

**Review by slinlee — CHANGES_REQUESTED** [review]
Just the comment on the auto tests, otherwise it looked good when I did a quick click-through.

**Comment by slinlee** [comment]
image
This is a nit, but we could auto-truncate the addresses using css so it doesn't break the layout for the rest of the text?

File: cypress/integration/walkthrough.js (hunk 2 above)
Comment (slavingia) [reply cypress/integration/walkthrough.js]: Fixed

**Comment by epan** [comment]
Good stuff @slavingia ! You handled a bunch of things I had on my mind

**Pull request merged**